### ISSUE TITLE
deepFM issue with ml-1m dataset

### ISSUE THREAD

**Issue by xray1111** [Leavingseason/OpenLearning4DeepRecsys#4]
Hi, I met with a strange problem while training deepFM model using ml-1m dataset: if enabling "is_use_fm_part" flag to True, the training process won't converge and the loss will become bigger and bigger. But if switching the flag off, just using dnn only, it seems ok. I only change the deepFM.py a little: For comparing the predicted rating with the GT value, I removed the softmax activation function for the last layer, and then output rmse error instead of auc.

**Comment by Leavingseason** [comment]
i am not sure if you can receive this message, but i am on a travel to Melbourne now, unfortunately no wifi access. I have to check the code next weekend when i return China

发自我的 iPhone

在 2017年8月17日，02:23，xray1111 <notifications@github.com<mailto:notifications@github.com>> 写道：


Hi, I met with a strange problem while training deepFM model using ml-1m dataset: if enabling "is_use_fm_part" flag to True, the training process won't converge and the loss will become bigger and bigger. But if switching the flag off, just using dnn only, it seems ok. I only change the deepFM.py a little: For comparing the predicted rating with the GT value, I removed the softmax activation function for the last layer, and then output rmse error instead of auc.

**Comment by Leavingseason** [comment]
@xray1111  hey  I am back now. Have you resolved the problem?

**Comment by xray1111** [comment]
@Leavingseason Sorry for late response, that problem may caused by a wrong when calcuating RMSE, I changed the code then it's fine. Thanks!

**Comment by nitinsurya** [comment]
Hi @xray1111 , could you share the code to run the deepFM with ml-1m dataset. I am a unclear on how to create the values.
